wca-regulations.md

```diff
@@ -380,7 +380,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A3a) The competitor may inspect the puzzle at the start of each attempt.
         - A3a1) The competitor is allotted a maximum of 15 seconds to inspect the puzzle and start the solve.
     - A3b) The judge prepares the timer by turning it on and resetting if necessary. Separately, the judge also prepares a stopwatch for timing inspection.
-        - A3b1) When the judge believes the competitor is ready, the judge asks "READY?". The competitor must be ready to start the attempt within one minute of being called, else the competitor forfeits the attempt (DNS), at the discretion of the judge.
+        - A3b1) When the judge believes the competitor is ready, the judge asks "READY?". The competitor must be ready to start the attempt within one minute of being called, or else the competitor forfeits the attempt (DNS), at the discretion of the judge.
```

Comment:
I'm not sure about this "or", maybe I'm too used to use "else" without putting or :P

Reply:
According to a discussion (https://able2know.org/topic/376948-1): 'else' used alone without a preceding 'or' is somewhat old-fashioned and non-standard.

As a programmer, using the word "else" by itself certainly becomes something that feels normal, so I know where you might be coming from.

However, I don't feel particularly strong about this change, so I won't be offended if you guys think it sounds better without the word "or".